src_c/surface.c

```diff
@@ -1627,8 +1634,8 @@ surf_get_alpha(pgSurfaceObject *self, PyObject *args)
     if (SDL_GetSurfaceBlendMode(surf, &mode) != 0)
         return RAISE(pgExc_SDLError, SDL_GetError());
 
-    if (mode != SDL_BLENDMODE_BLEND)
-        Py_RETURN_NONE;
+    if (mode == SDL_BLENDMODE_NONE)
```

Comment:
Because this wasn't in 1.9.6, I think we should keep compatibility of returning None when there is no alpha.